server/config/constants.js

```diff
@@ -29,8 +29,8 @@ module.exports = {
        "TELEMMETRY": "/action/data/v3/telemetry",
        "FORM_READ": "/action/data/v1/form/read",
        "USER_SEARCH": "action/user/v1/search/*",
-       "USER_ROLE": "/api/role",
-       "USERS": "/api/users",
+       "USER_ROLE": "/api/user/v1/role",
```

Comment:
What about front-end changes for this? 

@vaibhavbhuva of these changes are done, share the PR.

Reply:
@HarishGangula Please check

Reply:
@vaibhavbhuva  This is fine

Reply:
@HarishGangula What about front-end changes for this?  We need to update the same path in the fronted application  also right ?